test/index.js

```diff
@@ -143,4 +143,42 @@ describe("webpack-virtual-modules", function() {
       }
     });
   });
+
+  it('should work with path which parent dir not exists', function (done) {
+    var plugin = new Plugin({
+      'entry.js': 'const a = require("a").default; const b = require("b").default; export default a + b;',
+      'node_modules/a.js': 'export default 1;',
+      'node_modules/b.js': 'export default 2;'
+    });
+    var compiler = webpack({
+      context: __dirname,
+      plugins: [plugin],
+      entry: './entry.js',
+      output: {
+        path: path.resolve(__dirname),
+        filename: 'bundle.js',
+        library: 'test',
+        libraryTarget: 'umd'
+      },
+      target: 'node'
+    });
+    const fileSystem = new MemoryFileSystem();
+    compiler.outputFileSystem = fileSystem;
+    compiler.run(function(err, stats) {
+      assert(!err);
+      assert(!stats.hasErrors(), stats.toJson().errors[0]);
+      const outputPath = path.resolve(__dirname, 'bundle.js');
+      const outputFile = fileSystem.readFileSync(outputPath).toString();
+      const output = requireFromString(outputFile, outputPath).default;
```

Comment:
Why not just use `eval` here?

Reply:
just copy the code from my another project. and I want simulate the require function.

Reply:
@vipcxj Lets use `eval` here, it would result in simpler code

Reply:
I have tried `eval(outputFile)`, it not return the export value, it returns true, perhaps need more webpack config.

Reply:
@vipcxj Try this:
`const output = eval(outputFile + 'module.exports;').default;`